index.html

```diff
@@ -49,10 +49,12 @@ <h3>Breath of The Wild Weapons</h3>
           </div>
           <div class="column-half">
             <div class="row green-card-details border-radius-and-shadow">
-              <p class="font-size-22 margin-left">Description <br> <span class="font-size-15 description"></span></p>
-              <p class="font-size-22 margin-left">Common Locations <br> <span class="font-size-15 common-locations"></span></p>
-              <div class="row justify-end width-100 align-center">
-                <button class="save-button">Save</button>
+              <p class="font-size-22 margin-left-10">Description <br> <span class="font-size-15 description"></span></p>
+              <p class="font-size-22 margin-left-10">Common Locations <br> <span class="font-size-15 common-locations"></span></p>
+              <div class="row justify-end width-100 align-items-end">
+                <form class="row justify-end width-100 margin-bottom-5 margin-right-0">
+                  <button class="save-button">Save</button>
+                </form>
```

Comment:
Buttons don't necessarily need to be inside of forms. But if you have inputs like input, textarea, radio, etc. Those should generally be inside of a form element.